core/i18n/src/ITwinLocalization.ts

```diff
@@ -51,25 +50,50 @@ export class ITwinLocalization implements Localization {
       caches: [],
       ...options?.detectorOptions,
     };
+
     this._initOptions = {
       interpolation: { escapeValue: true },
       fallbackLng: "en",
+      maxRetries: 1, // prevents wasted time and potential timeouts when requesting localization files throws an error
       backend: this._backendOptions,
       detection: this._detectionOptions,
       ...options?.initOptions,
     };
 
     this.i18next
       .use(options?.detectorPlugin ?? i18nextBrowserLanguageDetector)
-      .use(options?.backendPlugin ?? XHR)
+      .use(options?.backendPlugin ?? Backend)
       .use(TranslationLogger);
   }
 
   public async initialize(namespaces: string[]): Promise<void> {
+    let initNamespaces: readonly string[];
+    let combinedNamespaces: string[];
+    let defaultNS: string;
+
+    if (this._initOptions.ns === undefined) {
```

Comment:
Could clean this up like so:

```ts
const initNamespaces = [this._initOptions.ns || []].flat() 
```

- The || replaces the undefined check
- Enclosing the `string|string[]` in an array and then calling `flat()` will ensure a 1-dimensional array.

Reply:
That is a lot nicer... I've made that change.